deploy/Dockerfile

```diff
@@ -18,8 +18,8 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/*
 
 RUN npm install --global yarn n
-# Upgrade to Node 14, above installs 10.
-RUN n 14
+# Upgrade to Node 18, above installs 12.
```

Comment:
Optional/bonus: Could we install 18 in the `RUN apt-get update` step instead? Maybe something like `nodejs=18.18.2` ([source](https://askubuntu.com/a/428778)). This line seems like an unnecessary hack.

Reply:
Unfortunately this would require changing our Linux distro version.  `debian bullseye` only has v12, AFAICT.

```
root@7e8a7da9e9f9:/# apt-cache policy nodejs
nodejs:
  Installed: (none)
  Candidate: 12.22.12~dfsg-1~deb11u4
  Version table:
     12.22.12~dfsg-1~deb11u4 500
        500 http://deb.debian.org/debian bullseye/main arm64 Packages
        500 http://deb.debian.org/debian-security bullseye-security/main arm64 Packages
```